tools/format.py

```diff
@@ -12,10 +11,18 @@ def register(command):
 
 def run(**kwargs):
 
-    # Get our relevant directories
-    for d in [os.path.join(b.project_dir, p) for p in ["module", "shared", "tests"]]:
-        for dirpath, dnames, fnames in os.walk(d):
-            for f in fnames:
-                if f.endswith((".h", ".c", ".cc", ".cxx", ".cpp", ".hpp", ".ipp", ".proto")):
-                    print("Formatting", f)
-                    call(["clang-format-6.0", "-i", "-style=file", os.path.join(dirpath, f)])
+    # Change into the project directory
+    os.chdir(b.project_dir)
+
+    # Use git to get all of the files that are committed to the repository
```

Comment:
Check for binaries before trying to format the files, or run in docker

Reply:
Running in docker!